style.css

```diff
@@ -0,0 +1,156 @@
+/* navbar */
+
+.navbar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 20px;
+  background-color: #ffff;
+  color: #6070ff;
+}
+
+.logo {
+  position: static;
+  width: 76px;
+  height: 20px;
+  left: 12px;
+  top: 10px;
+  font-family: 'poppins', sans-serif !important;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 18px;
+  line-height: 20px;
+
+  /* identical to box height, or 111% */
+  display: flex;
+  align-items: center;
+  text-align: center;
+}
+
+.body-bar {
+  background-image: url(./images/background.png);
+  position: absolute;
+  width: 462px;
+  height: 587.15px;
+  left: -34px;
+  top: 104px;
```

Comment:
Great using the image as `background-image` 👍. Let's try to make it look like the Figma design. Kindly expect `background-image` let's remove everything and use:
`background-size: contain;` 
`background-repeat: no-repeat;` that makes the image look great.

Reply:
Done but this kind off breaks the dimensions but it works well